packages/eslint-config/eslint-config.json

```diff
@@ -2,41 +2,5 @@
   "extends": [
     "@endo",
     "plugin:@jessie.js/recommended"
-  ],
-  "rules": {
-    "jsdoc/no-multi-asterisks": "off",
-    "jsdoc/multiline-blocks": "off",
-    // Use these rules to warn about JSDoc type problems, such as after
-    // upgrading eslint-plugin-jsdoc.
-    // Bump the 1's to 2's to get errors.
-    // "jsdoc/valid-types": 1,
-    // "jsdoc/no-undefined-types": [1, {"definedTypes": ["never", "unknown"]}],
-    "jsdoc/tag-lines": "off",
-    "import/no-extraneous-dependencies": [
-      "error",
-      {
-        "devDependencies": [
-          "**/*.config.js",
-          "**/*.config.*.js",
-          "**/*test*/**/*.js",
-          "**/demo*/**/*.js",
-          "**/scripts/**/*.js"
-        ]
-      }
-    ],
```

Comment:
I thought that dapps depended on `@agoric/eslint-config`. If we remove this, won't dapps lose their default config?